src/screens/Profiles/UserLibrary/components/UserLibraryScreen/styles.js

```diff
@@ -1,30 +1,63 @@
+import { StyleSheet } from 'react-native';
 import * as colors from 'kitsu/constants/colors';
+import * as constants from './constants';
 
 export const styles = {
```

Comment:
Ah yeah, I did this at 3am and didn't realize why stylesheet wasn't working (is because of native base) going to try and refactor native base out of this component